coderedcms/models/page_models.py

```diff
@@ -420,17 +421,22 @@ def __init__(self, *args, **kwargs):
             self.index_order_by = self.index_order_by_default
             self.index_show_subpages = self.index_show_subpages_default
 
+
     @classmethod
-    def get_edit_handler(cls):
-        """
-        Override to "lazy load" the panels overriden by subclasses.
-        """
-        return TabbedInterface([
+    def get_edit_handler_object_list(cls):
```

Comment:
I'm just curious on the reason for breaking this up into 2 methods.

Reply:
It would be clearer when I have the documentation examples up.  But, it doesn't make sense for every `Page` type to have an integrations panel.  Breaking this up into two functions allows page types to append their own panels to the list of panels rather than rewriting the whole function for panels.

Reply:
Ah I see. Another option might be to just add another panel, that only gets added when it's not empty. E.g.
``` python
if len(cls.integrations_panel) > 0:
    ObjectList(cls.integrations_panel, heading='Integrations', classname="integrations"), # classname corresponds to css class that could add the puzzle piece icon
```
Then on the concrete class:
``` python
integrations_panels = [ ... ]
```